java-components/build-request-processor/src/test/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommandTest.java

```diff
@@ -120,6 +121,7 @@ public void testModelResolver1()
     }
 
     @Test
+    @Ignore
```

Comment:
Why is this test not working and being excluded now?

Reply:
Initially I disabled recursive scanning, but then figured out a way to do it without incurring the memory cost. I have re-enabled it.